presto-hive-hadoop2/pom.xml

```diff
@@ -82,7 +82,7 @@
 
         <dependency>
             <groupId>org.alluxio</groupId>
-            <artifactId>alluxio-shaded-client</artifactId>
```

Comment:
core-common is a pretty different dependency than shaded client? Why did you update to this one instead of core-client-fs/hdfs?

Reply:
Sorry I commented on the wrong side of the diff. I'm referring to the line here now being core-common: https://github.com/prestodb/presto/blob/6e0fdf856004fbb30d9b1b93ea0db703cd25dfaf/presto-hive-hadoop2/pom.xml#L85

Reply:
Digging a bit into why `alluxio-shaded-client` is a dependency of` hive-hadoop2` we found #13743.
We believe this is no longer required, since `alluxio` dependencies will be already present in the `hive`, `iceberg`, `hudi` and `delta` connectors due to the `presto-cache` dependencies.

Reply:
I am okay with removing it then